### PR TITLE
Add SSS restoring files for E3SM-ARRM to buildnml

### PR DESCRIPTION
**Pull request by milenaveneziani**
Add the SSS restoring files for the two E3SM-ARRM configurations we currently have (ARRM60to10 and ARRM60to6).

### CODE REVIEW

**Comment by milenaveneziani** [comment]
@jonbob: this is a very simple PR, as we had forgotten to change the buildnml file to add the ARRM sss restoring filenames.

This was tested successfully on IC: Gennaro (@111-1000) was able to download all the necessary data (including the JRA stuff) without problems and has run one case for 6 months.
I am now setting up a new case/test on edison.

**Comment by rljacob** [comment]
Is there a test that uses these files?

**Comment by milenaveneziani** [comment]
We just ran a simulation test case with `res = TL319_oARRM60to10` and `compset = GMPAS-JRA`, and everything worked fine (new data was downloaded from public repo, and simulation ran fine for 6 months).

**Comment by rljacob** [comment]
I meant a test in the test suite.

**Comment by milenaveneziani** [comment]
I figured. But no, not that I am aware.

File: components/mpas-ocean/cime_config/config_component.xml [hunk 1]
```diff
@@ -122,7 +122,7 @@
  </entry>
 
   <description>
-    <desc compset="_MPASO">mpas ocean default:</desc>
+    <desc compset="_MPASO">MPAS ocean default</desc>
```
Comment (jonbob) [inline components/mpas-ocean/cime_config/config_component.xml]: I think the colon is consistent with the other components - why are we removing it with this PR?

File: components/mpas-seaice/cime_config/config_component.xml [hunk 2]
```diff
@@ -68,7 +68,7 @@
  </entry>
 
   <description>
-    <desc compset="_MPASCICE">MPAS seaice:</desc>
+    <desc compset="_MPASCICE">MPAS seaice</desc>
```
Comment (jonbob) [inline components/mpas-seaice/cime_config/config_component.xml]: same question here

File: cime/src/components/data_comps/datm/cime_config/config_component.xml [hunk 3]
```diff
@@ -10,7 +10,7 @@
        This file may have atm desc entries.
   -->
   <description modifier_mode="1">
-    <desc atm="DATM[%QIA][%WISOQIA][%CRU][%CRUv7][%GSWP3v1][%CPLHIST][%1PT][%NYF][%IAF][%JRA]"> Data driven ATM </desc>
+    <desc atm="DATM[%QIA][%WISOQIA][%CRU][%CRUv7][%GSWP3v1][%CPLHIST][%1PT][%NYF][%IAF][%JRA]">Data driven ATM </desc>
```
Comment (jonbob) [inline cime/src/components/data_comps/datm/cime_config/config_component.xml]: I don't think this should be part of this PR

File: components/mpas-ocean/cime_config/config_component.xml (hunk 1 above)
Comment (111-1000) [reply components/mpas-ocean/cime_config/config_component.xml]: Jon,

here is how the output looks right now:
...
ATM component is Data driven ATM interannual JRA55 forcing
LND component is Stub land component
ICE component is MPAS seaice
OCN component is MPAS ocean default
ROF component is Data runoff modelJRA55 interannual forcing
GLC component is Stub glacier (land ice) component
WAV component is Stub wave component
ESP component is 
...

and how it looked before:
...
ATM component is  Data driven ATM interannual JRA55 forcing
LND component is Stub land component
ICE component is MPAS seaice:
OCN component is mpas ocean default:
ROF component is Data runoff modelJRA55 interannual forcing
GLC component is Stub glacier (land ice) component
WAV component is Stub wave component
ESP component is 
...

so, no, the colon in not consistent with the other components and neither is the extra space ;-)

Comment (jonbob) [reply components/mpas-ocean/cime_config/config_component.xml]: OK, good catch then -- and thanks! But please separate this into two PR's then -- one to get the ARRM configurations working and one to clean up output

**Comment by jonbob** [comment]
@rljacob - we can add a test to either the extra-coverage or hi-res suites, but this configuration is somewhere between the low-res and high-res as far as required resources

**Comment by milenaveneziani** [comment]
@jonbob: there are only the ARRM changes now.

**Comment by jonbob** [comment]
merged to next

**Comment by milenaveneziani** [comment]
thanks @jonbob

**Comment by jonbob** [comment]
merged to master

**Pull request merged**